snippets/Google.Datastore.V1Beta3.Snippets/DatastoreClientSnippets.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using System.Linq;
+using Xunit;
+using static Google.Datastore.V1Beta3.CommitRequest.Types;
+using static Google.Datastore.V1Beta3.Key.Types;
+using static Google.Datastore.V1Beta3.PropertyFilter.Types;
+using static Google.Datastore.V1Beta3.ReadOptions.Types;
+
+namespace Google.Datastore.V1Beta3.Snippets
+{
+    [Collection(nameof(DatastoreSnippetFixture))]
+    public class DatastoreClientSnippets
+    {
+        private readonly DatastoreSnippetFixture _fixture;
+
+        public DatastoreClientSnippets(DatastoreSnippetFixture fixture)
+        {
+            _fixture = fixture;
+        }
+
+        [Fact]
+        public void Lookup()
+        {
+            var projectId = _fixture.ProjectId;
+            var key1 = _fixture.PrideAndPrejudiceKey;
```

Comment:
it seems like the appropriate way to form keys is a big reason that people would look at this sample